src/Utilities/Compiler/PooledObjects/PooledSortedSet.cs

```diff
@@ -59,5 +59,22 @@ public static PooledSortedSet<T> GetInstance(IComparer<T>? comparer = null)
             Debug.Assert(instance.Count == 0);
             return instance;
         }
+
+        /// <summary>
+        /// Gets a pooled instance of a <see cref="PooledSortedSet{T}"/> with an initializer and an optional comparer.
+        /// </summary>
+        /// <param name="initializer">Initial values for the set.</param>
+        /// <param name="comparer">Comparer to use, or null for the element type's default comparer.</param>
+        /// <returns>An empty <see cref="PooledSortedSet{T}"/>.</returns>
+        public static PooledSortedSet<T> GetInstance(IEnumerable<T> initializer, IComparer<T>? comparer = null)
```

Comment:
Do you need this method? I think this might have likely came when you did a merge of master sources? If so, I think you can revert this fine.

Reply:
hm strange, I reset all changes from master merge, maybe merged way before that. Anyways i am not referencing this, will revert